other/field-types.md

```diff
@@ -0,0 +1,36 @@
+# Schema Field Type Reference
+
+Below you can find each of the schema field types available in Apostrophe with links to a complete description of its function, properties, and refence code to help you use them in your project. For more general information on using schemas in Apostrophe, see the [Schema Guide](/tutorials/schema-guide/schema-guide.md). For more information on the usage of specific properties, see the [Schema Field Properties reference](field-properties.md).
+
+| Field | Description |
+|-------|-------------|
+|[area](/other/field-types/area.md) | an editable content area |
+|[array](/other/field-types/array.md) | A field which can store one of more objects as an array |
+|[attachment](/other/field-types/attachment.md) | provides access to upload a file to the server |
+|[boolean](/other/field-types/boolean.md) | provides a "True" or "False" input |
+|[checkboxes](/other/field-types/checkbox.md) | provides an array of checkboxes |
+|[color](/other/field-types/color.md) | provides a color picker | 
+|[date](/other/field-types/date.md) | provides a date picker |
+|[email](/other/field-types/email.md) | accepts a valid email address  |
+|[float](/other/field-types/float.md) | accepts input of a decimal number with stepped parameters |
+|[integer](/other/field-types/integer.md) | accepts input of an whole number |
+|[joinByArray](/other/field-types/joinByArray.md) | expresses a one-to-many relationship between an object and an array |
```

Comment:
"Between this document and one or more pieces or pages"

Reply:
Updated the description.